pyproject.toml

```diff
@@ -1,3 +1,69 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "flake8-pyi"
+dynamic = ["version"]
+authors = [
+  { name="Łukasz Langa", email="=lukasz@langa.pl" },
+]
+maintainers = [
+  { name="Jelle Zijlstra", email="jelle.zijlstra@gmail.com" },
+  { name="Alex Waygood", email="alex.waygood@gmail.com" },
+  { name="Sebastian Rittau" },
+  { name="Akuli" },
+  { name="Shantanu" },
+]
+description = "A plugin for flake8 to enable linting .pyi stub files."
+license = { text = "MIT" }
+readme = "README.md"
+requires-python = ">=3.7"
+keywords = [
+    "flake8",
+    "pyi",
+    "bugs",
+    "pyflakes",
+    "linter",
+    "qa",
+    "stubs",
+    "typing",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
```

Comment:
I kept this as it was, but we could maybe consider bumping it to "beta".

Reply:
Sure. Personally I don't get anything useful out of this classifier.